src/definitions/behaviors/api.js

```diff
@@ -334,7 +334,7 @@ $.api = $.fn.api = function(parameters) {
           cancelled: function() {
             return (module.cancelled || false);
           },
-          succesful: function() {
+          successful: function() {
```

Comment:
This will break the code because the function "was.successful" is already available in Line 341
See line 338
> module.verbose('This behavior will be deleted due to typo. Use "was successful" instead.');

Instead, you can completely remove the typo'ed function "was.succesful", as the next version 2.9.0 is supposed to be a minor release which allows for breaking changes.

Reply:
IOW, I should revert this change ?

Reply:
Either revert the change or, as already suggested, remove the whole function "successful". The correct spelled "successful" function is already there and used. The old spelled function was left in the code for backward compatibility

Reply:
Thanks for the clarification. Done in 9281165